ugs-platform/application/pom.xml

```diff
@@ -375,112 +350,6 @@
           </build>
         </profile>
 
-     <!-- Create the RaspberryPi application bundle -->
```

Comment:
Did you have a specific reason for removing this?

Reply:
Yes, the ordinary linux build works fine on RaspberryPI so I saw no reason to keep it.

Reply:
True, the current generic build runs everywhere since it doesn't include a JRE. If we have separate windows/linux/mac builds I don't think any of those would work with RPi since it uses an arm processor. Maybe I'm missing something?

Reply:
Ahh, you are right. I'll add it again, thanks!

Reply:
I reactivated this and are now using adoptopenjdk. An alternative would be to create dep-packages and depend on an JDK. But this is probably good enough.